tedium.js

```diff
@@ -47,29 +83,39 @@ Generate a token here:   https://github.com/settings/tokens
 
 const github = connectToGithub();
 
+const progressMessageWidth = 40;
+const progressBarWidth = 45;
 
 function getRepos() {
   const per_page = 100;
   const repos = [];
   const getFromOrg = promisify(github.repos.getFromOrg);
-  return Promise.resolve().then(() => {
-    function getReposFromPageOnward(page) {
-      return getFromOrg({org: 'PolymerElements', per_page, page}).then((results) => {
-        repos.push.apply(repos, results);
-        if (results.length === per_page) {
-          return getReposFromPageOnward(page + 1);
-        }
-        return repos;
-      });
-    }
+  const promises = [];
+  function getReposFromPageOnward(page) {
```

Comment:
It might be good to guard this with a sensible default value (or a suitable exception).